TrainTestSplit.py

```diff
@@ -3,7 +3,7 @@
 import json
```

Comment:
Which file needs to be run to create the JSON files?

Reply:
this is to store the final result as json file

Reply:
and also open testing_files.json